### PR TITLE
Use shell option when spawning `lerna exec`

### PR DESCRIPTION
**Pull request by jwb**
## Motivation and Context
The README example in the *exec* section would not supply any argument to npm view.

## How Has This Been Tested?
When I invoked the command properly, I discovered that environment variables are not expanded at all, so I enabled the shell option in the call to `spawn`.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by evocateur** [comment]
Can you add an [integration test](https://github.com/lerna/lerna/blob/master/test/integration/lerna-exec.test.js) for this?

File: src/commands/ExecCommand.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ export default class ExecCommand extends Command {
   getOpts(pkg) {
     return {
       cwd: pkg.location,
+      shell: true,
```
Comment (evocateur) [inline src/commands/ExecCommand.js]: I'm not sure this property is necessary, as [execa](https://github.com/sindresorhus/execa) is handling this for us.

Comment (evocateur) [inline src/commands/ExecCommand.js]: `lerna exec -- npm view \$LERNA_PACKAGE_NAME` works perfectly fine as-is on `master`, for example.

Comment (jwb) [inline src/commands/ExecCommand.js]: It was broken with spawn...

Comment (jwb) [inline src/commands/ExecCommand.js]: `npm view` reports on the package in the current directory.

Comment (evocateur) [inline src/commands/ExecCommand.js]: Ah yes, I just reproduced it locally.

```js
  test.concurrent("$LERNA_PACKAGE_NAME", () => {
    return initFixture("ExecCommand/basic").then((cwd) => {
      const args = [
        "exec",
        "echo",
        "\$LERNA_PACKAGE_NAME",
        "--concurrency=1",
      ];

      return execa(LERNA_BIN, args, { cwd }).then((result) => {
        expect(result.stdout).toMatchSnapshot("echo: $LERNA_PACKAGE_NAME");
      });
    });
  });
```

**Comment by jwb** [comment]
Added an integration test that fails on the master branch and passes with the fix.

**Comment by evocateur** [comment]
oh noes Windows :P

**Comment by jwb** [comment]
How should I make the test use a different value on WinDies?

**Comment by evocateur** [comment]
Actually, my windows env var syntax was wrong, it should be `%LERNA_PACKAGE_NAME%` apparently

File: src/commands/ExecCommand.js (hunk 1 above)
Comment (jwb) [inline src/commands/ExecCommand.js]: The integration test isn't evaluating the variables before sending the arguments to lerna.

**Comment by jwb** [comment]
Thanks for all the help @evocateur!

**Pull request merged**